src/data/reports/glass-data-submission/GLASSDataSubmissionDefaultRepository.ts

```diff
@@ -1034,7 +1063,7 @@ const emptyPage: PaginatedObjects<GLASSDataSubmissionItem> = {
 const moduleMapping: Record<string, string> = {
     AMC: "AMC",
     AMR: "AMR",
-    AMR_FUNGAL: "AMR - Fungal",
+    AMR_FUNGHI: "AMR - Fungal",
```

Comment:
I think that the name should be change to **AMR - Funghi** (we have changed all the names in glass-dev app)

Reply:
yes i had changed all the names from "Funghi" to "Fungal" in the report. but i notices that the module from the TEI api call return "AMR_FUNGHI" and not "AMR_FUNGAL" @anagperal

Reply:
aaah ok it was the other way around, then perfect!